shim/src/main.rs

```diff
@@ -240,3 +309,68 @@ fn main() -> Result<()> {
 
     process::exit(exit_code)
 }
+
+#[cfg(test)]
```

Comment:
We should either hook up the integration tests to hit the shim or really expand these tests to make sure we're able to parse valid turbo commands.

Reply:
Yeah this is just a start. I'll note that unless we have a parse bug around parsing a `bin` command, it shouldn't matter, as we don't do anything for other commands just yet, and we ignore all clap errors. Hence only tests for `bin` parsing. As we start to handle more cases in the Rust side, we should add more tests.